homeassistant/components/switch/ads.py

```diff
@@ -0,0 +1,74 @@
+"""Support for ADS switch platform."""
+
+import logging
+
+import voluptuous as vol
+from homeassistant.components.switch import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME
+from homeassistant.components.ads import DATA_ADS
+from homeassistant.helpers.entity import ToggleEntity
+import homeassistant.helpers.config_validation as cv
+
+
+_LOGGER = logging.getLogger(__name__)
+DEPENDENCIES = ['ads']
+DEFAULT_NAME = 'ADS Switch'
+CONF_ADSVAR = 'adsvar'
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_ADSVAR): cv.string,
+    vol.Optional(CONF_NAME): cv.string,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up switch platform for ADS."""
+    ads_hub = hass.data.get(DATA_ADS)
+    if not ads_hub:
+        return False
+
+    dev_name = config.get(CONF_NAME)
```

Comment:
Be consistent when naming variables, ie `name`.

Reply:
Done.